### PR TITLE
Use interpolation planner instead of sampling planner for hand

### PR DESCRIPTION
**Pull request by sjahr**
### Description

Small fix because I had issues with using the sampling-based planner for group hand when I changed it to PILZ or CHOMP.

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit2/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: doc/tutorials/pick_and_place_with_moveit_task_constructor/src/mtc_node.cpp [hunk 1]
```diff
@@ -116,6 +116,7 @@ mtc::Task MTCTaskNode::createTask()
   task.add(std::move(stage_state_current));
 
   auto sampling_planner = std::make_shared<mtc::solvers::PipelinePlanner>(node_);
+
```
Comment (sjahr) [inline doc/tutorials/pick_and_place_with_moveit_task_constructor/src/mtc_node.cpp:119]:
```suggestion
```

**Review by sea-bass — APPROVED** [review]
Yes! I

**Review by sea-bass — CHANGES_REQUESTED** [review]
Could you ensure the corresponding doc is also updated to reflect this?

File: doc/tutorials/pick_and_place_with_moveit_task_constructor/src/mtc_node.cpp (hunk 1 above)
Comment (sjahr) [inline doc/tutorials/pick_and_place_with_moveit_task_constructor/src/mtc_node.cpp:119]:
```suggestion

```

**Review by sea-bass — APPROVED** [review]

**Review by henningkayser — APPROVED** [review]

**Pull request merged**